crates/viewer/re_space_view_spatial/src/transform_component_tracker.rs

```diff
@@ -119,15 +124,39 @@ impl ChunkStoreSubscriber for TransformComponentTrackerStoreSubscriber {
             let entity_path_hash = event.chunk.entity_path().hash();
 
             for component_name in event.chunk.component_names() {
-                if self.transform_components.contains(&component_name) {
+                if self.transform_components.contains(&component_name)
+                    && event
+                        .chunk
+                        .components()
+                        .get(&component_name)
+                        .map_or(false, |list_array| {
+                            list_array
+                                .iter()
+                                .any(|list| !list.map_or(false, |list| list.is_empty()))
+                        })
```

Comment:
@teh-cmc is there a better way to do this?

Reply:
Hmm... maybe you could do something with the offsets instead, e.g.:
```rust
.map_or(false, |list_array| {
    list_array.offsets().lengths().any(|len| len > 0)
})
```
I think that's correct?

At the very least you can use `values_iter()` instead of `iter()`, which is going to be more performant in this case since you don't care about nullability (null arrays advertise a len of 0):
```rust
list_array.values_iter().any(|array| !array.is_empty())
```